hls-test-utils/hls-test-utils.cabal

```diff
@@ -46,6 +46,7 @@ library
     , lens
     , lsp-test                ^>=0.17
     , lsp-types               ^>=2.1
+    , safe-exceptions
```

Comment:
Seems you're not using anything from this package in this PR. Is it needed?

Reply:
I wanted to use `finally` from `safe-exceptions`, but missed the import `Control.Concurrent.Base`. Thanks for catching this!